deploy/kubernetes/elastic-agent-managed-kubernetes.yaml

```diff
@@ -135,6 +137,10 @@ spec:
         - name: sys-kernel-debug
           hostPath:
             path: /sys/kernel/debug
+        - name: elastic-agent-state
+          hostPath:
+            path: /var/lib/elastic-agent-managed/kube-system/state
```

Comment:
motivation for the path similar to the ECK -
`/var/lib` -  https://github.com/elastic/cloud-on-k8s/pull/5890#issuecomment-1197958554
`elastic-agent-managed/elastic-agent-standalone` - to not mix up 2 different options for the setup
`<namespace>` - to not overlap in case of multiple installations in different namespaces. This is hardcoded for now, but should be dynamically set if will be used helm/kustomize/etc

Reply:
Related to my comment below https://github.com/elastic/elastic-agent/pull/2550/files#r1178831027
Then for now we can put a comment again like:
#Update kube-system with the namespace of your agent installation
/var/lib/elastic-agent-managed/kube-system/state